docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -49,6 +49,16 @@ Breaking changes
   :py:meth:`~pvlib.pvsystem.PVSystem.calcparams_desoto` and
   :py:meth:`~pvlib.pvsystem.PVSystem.calcparams_cec` (:issue:`1118`, :pull:`1222`)
 
+* Switched the order of the outputs from the PSM3 iotools, notably
+  :py:func:`~pvlib.iotools.get_psm3` and :py:func:`~pvlib.iotools.read_psm3`
+  (:issue:`1245`, :pull:`1268`)
+
+* Changed the naming of the inputs `startdate`/`enddate` to `start`/`end` in
```

Comment:
```suggestion
* Changed the naming of the inputs ``startdate``/``enddate`` to ``start``/``end`` in
```

Reply:
@wholmgren What does double backtick do (if anything)? Or is it just a style convention?

Reply:
While markdown will render single backticks as code, sphinx/rst renders those as italics. Sphinx requires double backticks to render something as code. Most of the maintainers prefer using the code convention.